pkg/deployer/transformer/transformer.go

```diff
@@ -23,52 +23,52 @@ func init() {
 // It can change the Spinnaker configuration itself with TransformConfig.
 // It can also change the manifests before they are updated.
 type Transformer interface {
-	TransformConfig(hc *halconfig.SpinnakerConfig) error
-	TransformManifests(scheme *runtime.Scheme, hc *halconfig.SpinnakerConfig, gen *generated.SpinnakerGeneratedConfig, status *spinnakerv1alpha1.SpinnakerServiceStatus) error
+	TransformConfig() error
+	TransformManifests(scheme *runtime.Scheme, gen *generated.SpinnakerGeneratedConfig) error
 }
 
-// baseTransformer extends Transformer adding convenience methods.
-type baseTransformer interface {
-	transformServiceManifest(svcName string, svc *corev1.Service, hc *halconfig.SpinnakerConfig) error
-	transformDeploymentManifest(deploymentName string, deployment *v1beta1.Deployment, hc *halconfig.SpinnakerConfig) error
+// BaseTransformer extends Transformer adding convenience methods.
+type BaseTransformer interface {
+	transformServiceManifest(svcName string, svc *corev1.Service) error
```

Comment:
If `BaseTransformer` is now public, shouldn't its methods be also public so that code outside this package can implement them?